### PR TITLE
mfa level added

### PR DESCRIPTION
**Pull request by dhruvTor**
mfa level added @chaitanyapotti @metallicalfa2 please have a look

### CODE REVIEW

File: Sources/Web3Auth/Types.swift [hunk 1]
```diff
@@ -3,6 +3,14 @@ import UIKit
 import AuthenticationServices
 import SafariServices
 
+
+public enum MFALevel: String, Codable{
```
Comment (michaellee8) [inline Sources/Web3Auth/Types.swift:7]: whitespace here after `Codable` thx

**Review by michaellee8 — APPROVED** [review]
lgtm

**Comment by michaellee8** [comment]
@chaitanyapotti

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**